A32-xds-circuit-breaking.md

```diff
@@ -80,13 +86,21 @@ fail the call with status UNAVAILABLE. Such failed calls will not be retried,
 but they will be recorded to the `total_dropped_requests` counts in 
 cluster-level load reports and reported to the load reporting server.
 
+Note an EDS LB policy may be responsible for handling `service_name` switch
+used for endpoint discovery, requests sent to both the old and new services
+should be aggregated to the same counter and the total is restricted by the
+configured limit. Even though the failed requests will be recorded and reported
+in separate stats message.
+
 ## Implementation
 
 ### Java
-- Maintain a counter in the CDS LB policy for counting the number of 
+- Add a `max_requests` field in the EDS LB config. The value will be taken
```

Comment:
Please specify the exact protobuf changes for the EDS LB policy config.

Reply:
Sure. The proto change to the EDS LB policy config is general across languages, so I put it to the detailed design section.